taskcluster/client.py

```diff
@@ -490,6 +490,12 @@ def _makeHttpRequest(self, method, route, payload):
                     "Failed to establish connection",
                     superExc=rerr
                 )
+                
+            # Handles Non JSON responses
+            answer = response.get_content_type()
```

Comment:
Hmm, I'm not seeing that method in the requests library, are you able to link to the documentation?

http://docs.python-requests.org/en/master/user/quickstart/#response-headers seems to me to be a good way to get the value of the header.

We probably want to also trim whitespace from the header value before the comparison. 

We probably don't want to return `None` for all non-json responses, rather only when the status code is 204

Reply:
So what do you want me to return for non-json responses?

Reply:
For non-JSON responses, per [the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1135584#c8) the raw `response`  value is likely the best choice.

Reply:
Sure, thanks. I've changed the code to fit your review. I hope it is fine now. Requesting another review whenever you are free. Thanks!